go/ephemeral/team_ek_box_storage.go

```diff
@@ -261,8 +261,10 @@ func (s *TeamEKBoxStorage) DeleteExpired(ctx context.Context, teamID keybase1.Te
 		return nil, nil
 	}
 
+	merkleCtime := keybase1.TimeFromSeconds(merkleRoot.Ctime()).Time()
 	for gen, teamEKBox := range teamEKBoxes {
-		if ctimeIsStale(teamEKBox.Metadata.Ctime, merkleRoot) {
+		keyAge := merkleCtime.Sub(teamEKBox.Metadata.Ctime.Time())
+		if keyAge >= libkb.MinEphemeralKeyLifetime {
```

Comment:
Please add a comment here about why teamEKs never exceed the `MinEphemeralKeyLifetime`.